.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # dependencies
 /node_modules
+node_modules/
```

Comment:
Wondering why we need this. We don't have a `node_modules` in the `projects/atft` subdirectory according to the [package.json with empty dependencies](https://github.com/makimenko/angular-template-for-threejs/blob/master/projects/atft/package.json) therein.

(Note to self: absolute paths refer to the Git repo's root, had to look this up myself)